wordpress-mobile/WordPress-Android
# Add the view all tags & categories screen

This PR adds a new View all tags & categories screen.

To test:
1. Go to Stats
2. Scroll down to the Tags & Categories block
3. Tap View more
4. Notice a new screen opens up with more tags

Note: This PR can only be merged after the Comments view all screen is merged and the base branch is updated.

![image](https://user-images.githubusercontent.com/1522856/52449890-27a89280-2b39-11e9-8aad-6a3e6551ffd6.png)
